batch/Makefile

```diff
@@ -1,8 +1,10 @@
-.PHONY: hail-ci-build-image push-hail-ci-build-image
+.PHONY: hail-ci-build-image push-hail-ci-build-image test
```

Comment:
push-batch, push-batch-test, run, test-local, and deploy-batch should all be phony